docs/build.md

```diff
@@ -46,15 +57,16 @@ pre-built toolchains for Tier-3 targets.
 
 Check out Rust sources:
 
-```
-$ cd $MOTORH
-$ git clone https://github.com/rust-lang/rust.git
-$ cd rust
+```sh
+cd $MOTORH
+git clone git@github.com:rust-lang/rust.git
+cd rust
 ```
 
 Create `bootstrap.toml` file in $MOTORH/rust, as shown below:
 
-```
+```sh
+echo '
```

Comment:
This leaves an empty line at the top of the file; please use `echo '# bootstrap.toml...`